website/docs/secrets/manage-secrets-ui.mdx

```diff
@@ -31,13 +31,49 @@ Here, you will be prompted to enter the `External Secret Name` and the `Target K
 
 It's important to note that the chosen `SecretStore` may be a cluster-scoped `SecretStore` ie: `ClusterSecretStore` or a namespace-scoped `SecretStore`.
 
+If you choose a namespace scoped `SecretStore`, the new secret will be created on the same namespace as the `SecretStore`.
+
 ![Create new Secret](./imgs/create-secret-2.png)
 
-If you choose a namespace scoped `SecretStore`, the new secret will be created on the same namespace as the `SecretStore`.
+If you choose a cluster-scoped `ClusterSecretStore`, the new secret will be created in a namespace of your choice.
 
 ![Create new Secret](./imgs/create-secret-3.png)
 
-If you choose a cluster-scoped `ClusterSecretStore`, the new secret will be created in a namespace of your choice.
+#### Adding Secret Properties
+
+After you have choose your desired `SecretStore` & `SecretPath` the UI allows you to add secret properties in two different scenarios:
+
+##### 1. Adding and Removing Multiple Property Fields
+
+The first scenario allows you to add or remove multiple property fields. Each added `property` also has an optional `SecretKey` field. Here's how to do it:
+
+In the `Properties` section, click the `Add` button to create a new property field.
+
+![Add Property](./imgs/add-property-1.png)
+
+Enter the name of the `property` you want to create. You can add as many properties as you need.
+
+![Enter Property](./imgs/enter-property-1.png)
+
+If you wish to specify a `SecretKey` for the property, enter it in the `SecretKey` field. If this field is left blank, the `property` name will be used as the `SecretKey`.
+
+[Add SecretKey](./imgs/add-secret-key-1.png)
+
+To remove a property, click the `Remove` sign next to the property you wish to delete.
+
+Remember, this option allows you to have fine-grained control over which properties are included in your `ExternalSecret`.
+
+##### 2. Including All Properties
+
+The second scenario is to include all properties in your `ExternalSecret`. If the `Include all properties` checkbox is checked, all property inputs will be disabled and ignored, and all secrets including all keys under the specified `SecretPath` will be added. Here's how:
+
+Check the `Include all properties` checkbox. This will automatically disable the property input fields.
+
+![Include All Properties Checkbox](./imgs/include-all-properties-1.png)
+
+Using this option allows you to quickly create an `ExternalSecret` that includes all secrets under a specific `SecretPath`, without the need to specify each one individually. 
+
+Remember to use this option with caution.
```

Comment:
This is a great call out. It would be good to extend with the reasoning behind this caution for the user to understand the risks.  Given that is a warning, it might be good to use some https://docusaurus.io/docs/markdown-features/admonitions